### PR TITLE
Use correct primitive in archiveFile and handle limit

### PR DESCRIPTION
**Pull request by stevensoftware52**

### CODE REVIEW

File: crash-handler-process/platforms/util-win.cpp [hunk 1]
```diff
@@ -361,8 +369,8 @@ bool Util::archiveFile(const std::wstring& fileFullPath, const std::wstring& arc
 
 	// If we can't close the archive for some reason... does that mean we failed to archive the file??
 	// Probably it's archived successfully?? and then the archive is read/write stuck or something?? imo still return true
-	zipClose(zf, NULL);
-
+	zipClose(zf, NULL);
```
Comment (EddyGharbi) [inline crash-handler-process/platforms/util-win.cpp]: extra space

File: crash-handler-process/platforms/util-win.cpp [hunk 2]
```diff
@@ -341,9 +341,17 @@ bool Util::archiveFile(const std::wstring& fileFullPath, const std::wstring& arc
 
 	if (file.is_open())	{
 		file.seekg(0, std::ios::end);
-		long size = file.tellg();
+		uint64_t size = file.tellg();
 		file.seekg(0, std::ios::beg);
 
+		#undef max
```
Comment (EddyGharbi) [inline crash-handler-process/platforms/util-win.cpp:347]: why do we need to do that?

Comment (stevensoftware52) [reply crash-handler-process/platforms/util-win.cpp:347]: MSVS is using the max macro minwindef.h

File: crash-handler-process/platforms/util-win.cpp (hunk 1 above)
Comment (stevensoftware52) [reply crash-handler-process/platforms/util-win.cpp]: https://github.com/stream-labs/crash-handler/pull/79/commits/fd6988bd60fd128b116b26c3c204872a6dc86b51

**Review by EddyGharbi — APPROVED** [review]

**Pull request merged**